checkstyle.xml

```diff
@@ -42,7 +42,9 @@
     <property name="eachLine" value="true"/>
   </module>
 
-  <module name="NewlineAtEndOfFile" />
+  <module name="NewlineAtEndOfFile" >
+    <property name="lineSeparator" value="lf" />
```

Comment:
The other properties have no space before the final `/`.
So I would go with `value="lf"/>`

Reply:
Thanks 👍 , I fixed it

Reply:
@madhawa-gunasekara Could also fix the line above `"NewlineAtEndOfFile">`

Reply:
My mistake, I forgot to fix the NewlineAtEndOfFile.